src/index.ios.js

```diff
@@ -51,9 +55,16 @@ export default class Scribe extends Component {
     }
   }
 
-  updateUser(userAuth) {
+  updatePromptUN(value) {
     this.setState({
-      userAuth: userAuth
+      promptUN: value
+    });
+  }
+
+  updateUser(userAuth, username) {
```

Comment:
Can you also post to the server/database in this function? Posting messages doesn't work otherwise, even after choosing a display name.